### ISSUE TITLE
App uses permission GET_ACCOUNTS

### ISSUE THREAD

**Issue by eku** [TapiocaFox/Daijishou#639]
I could not find any explanation for the necessity of the right

```<uses-permission android:name="android.permission.GET_ACCOUNTS">```

Please explain or remove the right from the Andoid manifest.

**Comment by Jetup13** [comment]
I could be wrong but I believe is used for the Google Backup feature

> Daijishou > Settings > Backup and restore

**Comment by official-wizard** [comment]
@Jetup13 you're correct, this is for Google backups and doesn't need to be removed as it's a requirement for pulling Google Accounts to export/import backups, this is entirely **prompt based**. This can be closed!
